doc/user/content/release-notes.md

```diff
@@ -155,24 +147,37 @@ changes that have not yet been documented.
   SELECT t1.a FROM t1 JOIN t2 ON t1.a = t2.a GROUP BY t1.a
   ```
 
-- Support the [`WITH ORDINALITY`] modifier for function calls in the `FROM`
+- **Breaking change.** When using an arbitrary expression in an `ORDER BY` or
+  `DISTINCT ON` clause, only recognize references to input columns. Previously,
+  Materialize would recognize references to output columns as well.
+
+  See the [column references](/sql/select#column-references) section of the
+  `SELECT` documentation for details.
+
+- Fix a bug that could cause wrong results in queries that used the `ROWS FROM`
+  clause. The bug occurred if functions beyond the second function in the clause
```

Comment:
```suggestion
- Fix a bug that could cause wrong results in queries that used the `ROWS FROM`
  clause {{% gh 9654 %}}. The bug occurred if functions beyond the second function in the clause
```

Reply:
Just for posterity this one was actually an unreported bug, the fix for which was buried in #9686 (to which I've added a link). #9654 was a _different_ wrong results issue with `ROWS FROM` that shipped in the previous release.